Chunks/Chunk.cs

```diff
@@ -24,15 +24,18 @@ public class Chunk
 
     public readonly Mesh Mesh;
 
-    public Chunk(Vertex[] vertices, Vector3i position, Voxel[,,] voxels, bool createVao = true)
+    public readonly int Sphere;
+
+    public Chunk(Vertex[] vertices, Vector3i position, Voxel[,,] voxels, int sphere = 0, bool createVao = true)
     {
         Voxels = voxels;
         Position = position;
         Mesh = new Mesh(vertices, position, createVao);
+        Sphere = sphere;
```

Comment:
This is denotes which sphere the chunk belongs to. Having this as an int is handy because we can use this as an index in the array of sphereCenters (0 for the first & 1 for the second one)